CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v1.0.3 (2020-11-19)
+
+## OS Changes
+* Support setting Linux kernel parameters (sysctl) via settings ([#1158], [#1171])
+* Add `bpffs` mount to default installation ([#1160], **thanks @izaakschroeder!**)
```

Comment:
```suggestion
```
We definitely appreciate the communication, but there was no merge/change to the OS here, so I don't think we should call it out in the changelog.

Reply:
I missed that the PR was closed, not merged. Thanks.